dev/com.ibm.ws.logging.osgi/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -111,3 +111,6 @@ rollover.start.time.desc=The scheduled time of day for logs to first roll over.
 
 rollover.interval=Rollover interval for time based log rollover
 rollover.interval.desc=The time interval in between log rollovers, in minutes if a unit of time is not specified. Specify a positive integer followed by a unit of time, which can be days (d), hours (h), or minutes (m). For example, specify 5 hours as 5h. You can include multiple values in a single entry. For example, 1d5h is equivalent to 1 day and 5 hours. If rolloverStartTime is specified, the default value of rolloverInterval is 1 day. If both rolloverInterval and rolloverStartTime are unspecified, time based log rollover is disabled. Note: rolloverInterval does not apply to the console.log file.
+
+max.ffdc.age=Maximum age before an FFDC file is deleted.
+max.ffdc.age.desc=The maximum desired age before an FFDC file is deleted. Specify a positive integer followed by a unit of time, which can be days (d), hours (h), or minutes (m). For example, specify 2 days as 2d. You can include multiple values in a single entry. For example, 2d6h is equivalent to 2 days and 6 hours. Everyday at midnight, any FFDC file that has reached the maximum age will be deleted. By default, FFDC files will not be deleted based on file age.
```

Comment:
- Change "desired" to "wanted"
- Change "has reached" to "reaches"
- Change "will be deleted" to "is deleted"
- Change "will not be deleted" to "are not deleted"